maturity-level-two/src/Controllers/v1/CarController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Codit.LevelTwo.Extensions;
+using Codit.LevelTwo.Services;
+using Codit.LevelTwo.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using AutoMapper;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Net;
+using Codit.LevelTwo.Entities;
+
+namespace Codit.LevelTwo.Controllers.v1
+{
+    [Route("codito/v1/[controller]")]
+    [ApiController]
+    [ValidateModel]
+    public class CarController : ControllerBase
+    {
+        private readonly ICoditoRepository _coditoRepository;
+
+        public CarController(ICoditoRepository coditoRepository)
+        {
+            _coditoRepository = coditoRepository;
+            AutoMapperConfig.Initialize();
+        }
+
+        /// <summary>
+        /// Get all cars
+        /// </summary>
+        /// /// <param name="bodyType">Filter a specific body Type (optional)</param>
+        /// <remarks>Get all cars</remarks>
+        /// <returns>List of cars</returns>
+        [HttpGet(Name = Constants.RouteNames.v1.GetCars)]
+        [SwaggerResponse((int)HttpStatusCode.OK, "List of Cars")]
```

Comment:
@fgheysels  this will be part of the content negotiation guidelines. For now i mark as resolved, we will pick this up with the classic procedure :  Guideline > review > implementation